workspaces/src/network/account.rs

```diff
@@ -146,10 +147,14 @@ impl Contract {
         Self { account }
     }
 
+    /// Grab the current contract identifier
```

Comment:
maybe worth specifying that it's the current contract's account id. Contract identifier might be confusing to someone who thinks this is something else